pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -94,8 +94,12 @@ def __call__(self, value, system):
             cmd = ['pdftk', main.name]
             temp_files = [main]
             for url in pdf_to_join:
-                tmp_file = tempfile.NamedTemporaryFile(suffix='.pdf')
                 result = requests.get(url)
+                content_type = result.headers.get('content-type')
+                log.debug("document url: " + url + " => content_type: " + content_type)
+                if content_type != 'application/pdf':
+                    continue
```

Comment:
Continue without anything?
I think that we at least have a warning log ...

Reply:
ok I will introduce the log..

Reply:
done in the last push